pages/01.HDM Documentation/06.vcd/10.Network Planning/05.Cloud Network Config/default.md

```diff
@@ -56,20 +61,31 @@ This network will be created for HDM Appliance communications and for migrating
 
 ![alt_text](images/image9.png "image_tooltip")
 
-#### <a name="configure-vpn-tunnel-edge"> Step 3: Configure VPN tunnel with Edge Gateway </a>
+### <a name="configure-vpn-tunnel-edge"> Step 3: Configure VPN tunnel with Edge Gateway </a>
 
 Create an IPSec tunnel between the organization's edge gateway and the on-premises datacenter. To complete the VPN setting the following information will be required:
 
-*   Local ID - Edge gateway tenant external network IP 
-*   Local Endpoint - Edge gateway tenant external network IP
+*   Local ID - Edge gateway tenant external network IP (see _Figure 1_)
```

Comment:
http://ec2-34-222-41-154.us-west-2.compute.amazonaws.com/hdm%20documentation/vcd/network%20planning/cloud%20network%20config#step-3-configure-vpn-tunnel-with-edge-gateway

Public IP is not masked.

Reply:
double check, I have changed the public IP that is not our public IP. Can't mask the very thing we want to show.